.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Should we really keep this in the repo? Not all of schedar uses vscode.

Reply:
Ah yes, we have to remove that.